documentation/operations-manual.md

```diff
@@ -1 +1,36 @@
-# Travelcy Operations Manual
\ No newline at end of file
+# Travelcy Operations Manual
+
+## Overview
+Travelcy is a native android application developed in Kotlin with Android Studio.
+
+## Coding conventions
+We have been following Kotlins' standard coding convention, which can be found here https://kotlinlang.org/docs/reference/coding-conventions.html
+
+## Testing
+Tests are defined under seperate packages com.travelcy.travelcy (test) and com.travelcy.travelcy (androidTest), under which we define our tests, mocks of repositories/services etc.
+Creating tests is a necessary part of our workflow, and it is taken into consideration when reviewing pull requests.
+
+## Setting up the development environment
+1. Download and install Android Studio https://developer.android.com/studio/
+2. Set up git on your machine if you haven't already and clone the repository from here https://github.com/Travelcy/Travelcy
+3. You may have to download and install the most recent java sdk https://www.oracle.com/java/technologies/javase-jdk13-downloads.html
+
+## Pull request/review policy
+For every new feature or bug fix you should create a branch based on develop. When work is done you create a pull request in github and wait for it to be reviewed.
+
+When reviewing a pull request, there are certain standards that need to be met. Are coding conventions being met, have relevant tests been created, does everything build and run accordingly, is there redundant code?
+
+## CI/CD
```

Comment:
Would be good to have more detailed descriptions on what the different workflows actually to and what triggers them to be run, for example beta deployment to Google Play, automatic documentation generation for production releases, test artifacts etc.